spec/hallmonitor/outputters/dogstatsd_outputter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require 'hallmonitor/outputters/dogstatsd_outputter'
+
+module Hallmonitor
+  module Outputters
+    RSpec.describe DogstatsdOutputter do
+      let(:dogstatsd_client) { instance_double(Datadog::Statsd) }
+      let(:prefix) { 'test' }
+      let(:outputter) { described_class.new(prefix) }
+
+      before do
+        allow(dogstatsd_client).to receive(:namespace=)
+        allow(Datadog::Statsd).to receive(:new).and_return(dogstatsd_client)
+      end
+
+      it 'can be instantiated' do
+        expect { outputter }.to_not raise_error
+      end
+
+      context '#process' do
+        let(:event_name) { 'foo.bar.baz' }
+        let(:event_tags) { {tags: [{foo:"bar"}]} }
+        let(:event_tags_expected) {{tags:["tags:[{:foo=>\"bar\"}]"]}}
```

Comment:
That's really odd then, because this code: 

`@statsd.count(event_name, event.count, tags: process_tags(event.tags))`

shouldn't be producing `"foo.bar.baz", 1, {:tags=>["tags:[{foo:\"bar\"}]"]}`, right?  the result of `process_tags` should be an array I think